### PR TITLE
translation to croatian

### PR DESCRIPTION
**Pull request by sime2408**
## Pull Request Checklist

- [x] **Target branch:** Pull requests should target the `dev` branch.
- [x] **Description:** Briefly describe the changes in this pull request.
- [x] **Changelog:** Ensure a changelog entry following the format of [Keep a Changelog](https://keepachangelog.com/) is added at the bottom of the PR description.
- [x] **Documentation:** Have you updated relevant documentation [Open WebUI Docs](https://github.com/open-webui/docs), or other documentation sources?
- [x] **Dependencies:** Are there any new dependencies? Have you updated the dependency versions in the documentation?
- [x] **Testing:** Have you written and run sufficient tests for the changes?
- [x] **Code Review:** Have you self-reviewed your code and addressed any coding standard issues?

---

## Description

translation to croatian

---

### Added

- Language Enhancements**: Croatian translation added.

### Fixed

-

### Changed

- src/lib/i18n/locales/languages.json

### Removed

-

### Security

-

### Breaking Changes

-

---

### Additional Information

-

### CODE REVIEW

**Comment by Silentoplayz** [comment]
Please submit a new PR without any modifications made to `CHANGELOG.md`.

File: src/lib/i18n/locales/languages.json [hunk 1]
```diff
@@ -114,5 +114,9 @@
 	{
 		"code": "dg-DG",
 		"title": "Doge 🐶"
+	},
```
Comment (Silentoplayz) [inline src/lib/i18n/locales/languages.json:117]: Should be

```
},
	{
		"code": "hr-HR",
      	        "title": "Croatian"
	}
]
```

Comment (sime2408) [reply src/lib/i18n/locales/languages.json:117]: Hey, what do you mean here? I did format it json like, see below.

File: src/lib/i18n/locales/languages.json [hunk 2]
```diff
@@ -114,5 +114,9 @@
...
+	{
+		"code": "hr-HR",
+      	"title": "Croatian"
```
Comment (Silentoplayz) [inline src/lib/i18n/locales/languages.json:120]: `"title": "Croatian"` needs another... 8 spaces in front of it to be aligned with `"code": "hr-HR",`

**Comment by Silentoplayz** [comment]
LGTM!

**Comment by tjbck** [comment]
Thanks everyone!

**Pull request merged**